odoo/addons/test_new_api/models.py

```diff
@@ -71,7 +71,7 @@ class Discussion(models.Model):
     moderator = fields.Many2one('res.users')
     categories = fields.Many2many('test_new_api.category',
         'test_new_api_discussion_category', 'discussion', 'category')
-    participants = fields.Many2many('res.users')
+    participants = fields.Many2many('res.users', context={'active_test': False})
```

Comment:
fix the test instead

Reply:
We'll fix tests later. Changing tests to use 'admin' simply breaks many of them...